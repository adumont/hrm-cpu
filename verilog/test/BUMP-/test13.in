07 95
22 4a
02 7f
15 cc
29 9e
13 0a
14 c0
08 64
30 7e
59 ac
10 14
26 36
57 f5
