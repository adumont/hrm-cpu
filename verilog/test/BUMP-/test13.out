94
49
7e
cb
9d
09
bf
63
7d
ab
13
35
f4
